src/cmd/scan.rs

```diff
@@ -65,6 +67,7 @@ pub fn extract_data(
             return Err(format!("Error: failed to extract repository data: {}", err));
         }
     };
+    spin.stop_with_newline();
```

Comment:
In case of an error between `Spinner::new` (line 56) and here, `stop_with_newline()` is never called causing the lack of a newline after the Spinner message.

Reply:
Thank you for flagging that, I will take care of it after we have merged, https://github.com/elhmn/waking-git/pull/49 as it changes a few things on how we handle this spinners.